### DISCUSSION TITLE
Tailwindcss 2.0 Update?

### DISCUSSION THREAD

**Discussion by firdausbaihaqi** [moesaid/cleopatra#12] [Ideas]
Hi @moesaid, firstly I want to thank you for your great work on Cleopatra template, I really love it 😄!!

have you ever consider updating your tailwind version to tailwind 2.0? I see you are using version 1.4 instead. tailwind's newest version has the ability to control component responsive behavior much easier with their breakpoints classes, so I think that would make your contributor work easier if you update your tailwind version to the newest one.

**Comment by moesaid** [comment]
Hey @effendikiki ! , thanks for your kind words, I appreciated ❤️ 😃 ,

I did, and I still want to, but I am really fighting the time to get this done, I have a lot of on my plate, I am considering to build VueJs version with more advanced Componnetns so, when/if I did that I will defiantly use tailwind 2.0^
